{"id":"../../node_modules/axios/lib/helpers/isAxiosError.js","dependencies":[{"name":"C:\\WebApps\\natours\\package.json","includedInParent":true,"mtime":1651830721831},{"name":"C:\\WebApps\\natours\\node_modules\\axios\\package.json","includedInParent":true,"mtime":1651827082565},{"name":"./../utils","loc":{"line":3,"column":20},"parent":"C:\\WebApps\\natours\\node_modules\\axios\\lib\\helpers\\isAxiosError.js","resolved":"C:\\WebApps\\natours\\node_modules\\axios\\lib\\utils.js"}],"generated":{"js":"'use strict';\n\nvar utils = require('./../utils');\n\n/**\n * Determines whether the payload is an error thrown by Axios\n *\n * @param {*} payload The value to test\n * @returns {boolean} True if the payload is an error thrown by Axios, otherwise false\n */\nmodule.exports = function isAxiosError(payload) {\n  return utils.isObject(payload) && (payload.isAxiosError === true);\n};\n"},"sourceMaps":null,"error":null,"hash":"ce150bb3e1ac7cf1f14a7343dc09545e","cacheData":{"env":{}}}